### PR TITLE
Comment PHP variables in htaccess.

### PR DESCRIPTION
**Pull request by hemratna**

### CODE REVIEW

File: public/.htaccess [hunk 1]
```diff
@@ -20,7 +20,8 @@ Options +SymLinksIfOwnerMatch
     RewriteRule !^admin index.php?%{QUERY_STRING} [L]
 </ifModule>
 
-<IfModule mod_php7.c>
-  php_value upload_max_filesize 50M
-  php_value post_max_size 100M
-</IfModule>
\ No newline at end of file
+#Uncomment line to set the PHP variables
```
Comment (rijkvanzanten) [inline public/.htaccess]:
```suggestion
# Uncomment the following lines to increase the max upload size of Directus
```

File: public/.htaccess [hunk 2]
```diff
@@ -20,7 +20,8 @@ Options +SymLinksIfOwnerMatch
     RewriteRule !^admin index.php?%{QUERY_STRING} [L]
 </ifModule>
 
-<IfModule mod_php7.c>
-  php_value upload_max_filesize 50M
-  php_value post_max_size 100M
-</IfModule>
\ No newline at end of file
+# Uncomment the following lines to midify the PHP variables like increase the max upload size of Directus
```
Comment (rijkvanzanten) [inline public/.htaccess]: Haha I'm fine with ignoring my suggestion, but here's another one to fix the spelling/grammar error 😉 

```suggestion
# Uncomment the following lines to modify PHP settings. The lines below can be used to increase the max upload size of files in Directus
```

Comment (hemratna) [reply public/.htaccess]: @rijkvanzanten I am not ignoring your suggestions. I am really sorry if you feel that way.
 I want to make sure that comment shows, *Anyone can modify any available PHP variables from `.htaccess`*

Thanks a lot for correcting me. 🙂

Comment (rijkvanzanten) [reply public/.htaccess]: Haha I'm just playing with ya. No hard feelings 😘

**Pull request merged**